### ISSUE TITLE
loss design

### ISSUE THREAD

**Issue by rwn17** [VisualComputingInstitute/diffusion-e2e-ft#7]
Hi Authors,

Thanks for your excellent work and it's truly amazing! I noticed that you calculate the loss on decoded depth instead of latent space. I didn't find an ablation for such a design in the paper. I wondered whether such a design is crucial for enabling accurate single-step prediction. 

Thanks in advance and look forward to your training code!

Best,
Weining

**Comment by kabouzeid** [comment]
Yes, pixel-wise supervision is important for achieving the best results. However, it is not what makes single-step prediction possible; rather, it's the other way around: single-step prediction makes end-to-end fine-tuning the obvious choice.
